packs/spell-effects/aura-bane.json

```
@@ -0,0 +1,58 @@
+{
+    "_id": "F5NnpKkn4pKwEeFB",
+    "img": "systems/pf2e/icons/spells/bane.webp",
+    "name": "Aura: Bane",
+    "system": {
+        "badge": {
+            "labels": [
+                "10 ft.",
+                "20 ft.",
+                "30 ft.",
+                "40 ft.",
+                "50 ft.",
+                "60 ft.",
+                "70 ft.",
+                "80 ft.",
+                "90 ft.",
+                "100 ft."
+            ],
+            "loop": false,
+            "type": "counter",
+            "value": 1
+        },
+        "description": {
+            "value": "<p>Granted by @UUID[Compendium.pf2e.spells-srd.Item.Bane].</p><hr /><p>You fill the minds of your enemies with doubt. Enemies in the area must succeed at a Will save or take a –1 status penalty to attack rolls as long as they are in the area. Once per round on subsequent turns, you can Sustain the spell to increase the emanation's radius by 10 feet and force enemies in the area that weren't yet affected to attempt another saving throw.</p><hr /><p>@UUID[Compendium.pf2e.spell-effects.Item.Spell Effect: Bane]</p>"
+        },
+        "duration": {
+            "expiry": "turn-start",
+            "sustained": false,
+            "unit": "minutes",
+            "value": 1
+        },
+        "level": {
+            "value": 1
+        },
+        "publication": {
+            "license": "ORC",
+            "remaster": true,
+            "title": "Pathfinder Player Core"
+        },
+        "rules": [
+            {
+                "key": "Aura",
+                "radius": "+ @item.badge.value * 10"
```

```suggestion
                "radius": "@item.badge.value * 10"
```
I'm not big on tracking effects like this